src/modules/dashboard/components/TaskComments/TaskComments.tsx

```diff
@@ -72,26 +76,27 @@ const handleKeyboardSubmit = (
   return false;
 };
 
-const TaskComments = ({
-  taskTitle,
-  colonyAddress,
-  draftId,
-  history,
-}: Props) => {
-  const { username, walletAddress } = useLoggedInUser();
-
-  const onSuccess: OnSuccess = useCallback(
-    (result, { resetForm, setStatus }) => {
-      setStatus({});
-      // @ts-ignore proper formik typings!
-      resetForm({ comment: '' });
-    },
-    [],
-  );
-
-  const transform = useCallback(
-    mergePayload({ colonyAddress, author: walletAddress, draftId, taskTitle }),
-    [colonyAddress, draftId],
+const TaskComments = ({ draftId, history }: Props) => {
+  const { username } = useLoggedInUser();
+
+  const [sendComment] = useSendTaskMessageMutation();
+  const onSubmit = useCallback(
+    ({ comment: message }: FormValues, { resetForm }: FormikBag<any, any>) =>
```

Comment:
I think this can be a little more tightly typed:

```suggestion
    ({ comment: message }: FormValues, { resetForm }: FormikBag<object, FormValues>) =>
```